style.css

```diff
@@ -1125,4 +1155,254 @@ h4 {
     font-size: 0.8rem;
     width: -1rem;
   }
+
+  .popPicturesPreview {
+    max-width: 40vw;
+  }
+
+  .lists {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    gap: 0.5rem;
+    justify-content: left;
+    padding-left: 5rem;
+  }
+
+  .popProjectTitle {
+    margin-left: 5rem;
+    padding-bottom: 2rem;
+    font-family: "Poppins", sans-serif;
+    font-size: 2.8rem;
+    color: #1a2236;
+    font-weight: 900;
+  }
+
+  #crossTypo2 {
+    text-align: end;
+    padding: 2rem;
+    font-weight: 200;
+    cursor: pointer;
+  }
+}
+
+.inlineBlockButton {
+  display: block;
+}
+
+@media screen and (min-width: 768px) {
+  .inlineBlockButton {
+    display: flex;
+  }
+
+  .popButText {
+    font-family: "Poppins", sans-serif;
+    color: #1a2236;
+    font-weight: 600;
+    display: flex;
+    flex-direction: row;
+    text-align: center;
+    translate: -2rem -1rem;
+    cursor: pointer;
+  }
+
+  #popUnionButton {
+    background-image: url(./img/popup/unionPopup.svg);
+    background-repeat: no-repeat;
+    background-position-y: 0.7rem;
+    background-position-x: 20rem;
+    background-size: 9%;
+    width: 23rem;
+  }
+
+  #popGitHubButton {
+    background-image: url(./img/popup/GithubPopup.svg);
+    background-repeat: no-repeat;
+    background-position-y: 0.6rem;
+    background-position-x: 20rem;
+    background-size: 10%;
+    width: 23rem;
+    margin-left: 1rem;
+  }
+
+  .popupButtons {
+    width: 8rem;
+    outline: 2px solid #1a2236;
+    margin-bottom: 2rem;
+    background-color: #efefef;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+}
+
+#crossTypo {
+  text-align: end;
+  padding: 2rem;
+  font-weight: 200;
+  cursor: pointer;
+}
+
+.popupContainer2 {
+  height: 100vh;
+  display: none;
+  position: fixed;
+  flex-direction: column;
+  width: 100%;
+  background-color: rgb(26, 34, 54, 0.98);
+}
+
+.popupContainer {
+  width: 100vw;
+  height: 100%;
+  display: none;
+  position: fixed;
+  left: 0;
+  top: 0;
+  flex-direction: column;
+  background-color: rgba(255, 255, 255);
+}
+
```

Comment:
- [ ] kindly consider sticking to the provided figma design for the pop-up
- your current pop-up  design 👇
<img width="1317" alt="Screen Shot 2023-03-18 at 20 14 38" src="https://user-images.githubusercontent.com/60146030/226126369-a24442a9-94c3-4801-a0f6-657154b5e4bf.png">

- provided pop-up design 👇 
<img width="483" alt="Screen Shot 2023-03-18 at 20 16 45" src="https://user-images.githubusercontent.com/60146030/226126524-1fa0dc75-5dea-4e92-8aa2-3af873214737.png">

- all elements circled are missing on your pop-up design kindly consider adding them have the right design as provided and this will help you to full-fill the design requirements

1.when adding the close icon kindly consider making sure that When the user clicks (or taps) the close (X) button, the popup disappears.

2. after adding all the circled elements missing kindly consider The popup is responsive, it fits mobile and desktop sizes properly.